libraries/src/test/java/com/baeldung/circularfifoqueue/example/CircularFifoQueueTest.java

```diff
@@ -0,0 +1,161 @@
+package com.baeldung.circularfifoqueue.example;
```

Comment:
Let's move this out of the com.baeldung.circularfifoqueue.example package and just put it in com.baeldung.circularfifoqueue